packages/components/src/docs/General/6-Known Browser Issues.mdx

```diff
@@ -0,0 +1,28 @@
+import { Source, Markdown, Meta } from '@storybook/blocks';
+import playwrightVersion from 'playwright-version';
+import pkg from '../../../package.json';
+
+<Meta title="Docs/General/Known Browser Issues" />
+
+# Known Browser Issues
+
+We've identified several browser issues that may affect visual consistency while using Solid Components.
+It's important for developers to be aware of these issues when implementing our components.
+
+## Missing Border Radius for Focus Styles in Safari
+
+In Safari, focus styles may lack border radius, unlike other browsers. This
```

Comment:
Please provide concrete examples or say, that this affects every component.

Reply:
Removing the entire section as this problem seems to have been fixed on Safari's end.